gaiacli account <address> panics if address isn't found

If there has been no transaction to an account, `gaiacli account <address>` panics with the following error:
```
panic: UnmarshalBinaryBare cannot decode empty bytes

goroutine 1 [running]:
github.com/cosmos/cosmos-sdk/x/auth/client/cli.GetAccountDecoder.func1(0x0, 0x0, 0x0, 0x1, 0xc7d53b, 0x15, 0x0)
        /root/go/src/github.com/cosmos/cosmos-sdk/x/auth/client/cli/account.go:25 +0xd7
github.com/cosmos/cosmos-sdk/x/auth/client/cli.GetAccountCmd.func1(0xc42098f900, 0xc420998130, 0x1, 0x1, 0x0, 0x0)
        /root/go/src/github.com/cosmos/cosmos-sdk/x/auth/client/cli/account.go:56 +0x1a6
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).execute(0xc42098f900, 0xc420998100, 0x1, 0x1, 0xc42098f900, 0xc420998100)
        /root/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:762 +0x468
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x126d360, 0xc8c1e9, 0x24, 0xc4200373a8)
        /root/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra.(*Command).Execute(0x126d360, 0xc420960fa0, 0x126d450)
        /root/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/cosmos/cosmos-sdk/vendor/github.com/tendermint/tmlibs/cli.Executor.Execute(0x126d360, 0xdcd188, 0x2, 0xc420036f30)
        /root/go/src/github.com/cosmos/cosmos-sdk/vendor/github.com/tendermint/tmlibs/cli/setup.go:89 +0x4e
main.main()
        /root/go/src/github.com/cosmos/cosmos-sdk/cmd/gaia/cmd/gaiacli/main.go:123 +0x8a7
```
Lets just check if res is empty right before account.go line 56, with an error message something to the extent of "No node with address `<address>` found in the blockchain"
Thanks @kwunyeung for pointing out what caused this.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -8,6 +8,7 @@ FEATURES
 IMPROVEMENTS
 * export command now writes current validator set for Tendermint
 * [tests] Application module tests now use a mock application
+* [gaiacli] Fix error message when account isn't found when running gaiacli account
 
 FIXES
 * [lcd] Switch to bech32 for addresses on all human readable inputs and outputs

--- a/x/auth/client/cli/account.go
+++ b/x/auth/client/cli/account.go
@@ -52,6 +52,12 @@ func GetAccountCmd(storeName string, cdc *wire.Codec, decoder auth.AccountDecode
 				return err
 			}
 
+			// Check if account was found
+			if res == nil {
+				return sdk.ErrUnknownAddress("No account with address " + addr +
+					" was found in the state.\nAre you sure there has been a transaction involving it?")
+			}
+
 			// decode the value
 			account, err := decoder(res)
 			if err != nil {

